Contributing.md

```diff
@@ -9,3 +9,228 @@ If you've built a new reporting type or added a feature, feel free to open a pul
 If you're encountering an issue getting a report to work, it's possible that it has't been tested against the specific iOS version that created the backup. 
 
 When adding an issue for a bug report, be sure to include the iOS version that causes it.
+
```

Comment:
Does this whole block belong to the contributing document? Feels to me like very specific documentation around how it works and why, right?

Reply:
Initially, the repository is a fork. For now, we don’t plan to change the way the documentation is structured.